TalkAloud/TalkAloud/ViewControllers/AudioRecordingsViewController.swift

```diff
@@ -49,6 +56,16 @@ class AudioRecordingsViewController: UIViewController {
         recordingsTableView.register(UINib(nibName: "TagCollectionViewTableViewCell", bundle: nil), forCellReuseIdentifier: "TagCell")
     }
     
+    private func updateTableView(isEmptyTableView: Bool) {
```

Comment:
You don't need this method. You can directly control the state of the label in the property observer logic.